.github copy/list.rb

```diff
@@ -0,0 +1,17 @@
+require  'enumerable'
+
+class List
+    include  enumerable
+
+    def initialize(*list)
+        @list = list
+    end
+
+    def each 
+        i = 0
+        while i < @list.length
+            yield @list[i]
+            i += 1
+        end
+    end
```

Comment:
- [ ] Optional suggestions

You should return a normal Enumerator using to_enum, so I would recommend adding the below line before I declaration

- `return to_enum(:each) unless block_given?` and return @list after end of while.

Reply:
it is implemented, thank you for the suggestion!